src/utils/tools/getThemeDir.ts

```diff
@@ -0,0 +1,22 @@
+import { iconsTheme, iconsThemeV1 } from 'common/types'
+import { ThemesDirectories } from 'common/constants'
+
+// *********************************
+// This function is used to return the appropriate directory, based on the theme:
+// we're handling the case of `svg` and `svg-outlined` manually for now, until the full
+// depreciation of version 1 APIs
+// *********************************
+
+const getThemeDir = (theme: iconsTheme | iconsThemeV1 = iconsTheme.filled) => {
+  let themeDir: string
+  if (theme === 'svg') {
+    themeDir = ThemesDirectories[iconsTheme.filled]
```

Comment:
one question. why is this needed in the first place? if we provide the enums filled, outlined values of `svg & svg-outlined` resp, we can use it directly without these conditions right?

Reply:
This's just a backward compatibility, `v1` apis are using `svg` and `svg-outlined`, as per our conversation #12, we're using `filled` and `outlined`, that's way we're just mapping the old values to use the directories we specified in `filled` and `outlined` until the full deprecation of `v1`.

Have I answered this question? i'm not sure if I understood your question correctly.